adblock-lean

```diff
@@ -529,25 +539,25 @@ restore_saved_blocklist()
 
 clean_dnsmasq_dir()
 {
-	rm -f /tmp/dnsmasq.d/.blocklist.gz /tmp/dnsmasq.d/blocklist /tmp/dnsmasq.d/conf-script /tmp/dnsmasq.d/.extract_blocklist
+	rm -f /var/run/adblock-lean/dnsmasq.d/.blocklist.gz /var/run/adblock-lean/dnsmasq.d/blocklist /var/run/adblock-lean/dnsmasq.d/conf-script /var/run/adblock-lean/dnsmasq.d/.extract_blocklist
 }
 
 import_blocklist_file()
 {
-	[[ -f /tmp/blocklist ]] || return 1
+	[[ -f /var/run/adblock-lean/blocklist ]] || return 1
 
 	if [[ "${compress_blocklist}" == 1 ]]
 	then
 		clean_dnsmasq_dir
-		printf "conf-script=\"busybox sh /tmp/dnsmasq.d/.extract_blocklist\"\n" > /tmp/dnsmasq.d/conf-script
-		printf "busybox gunzip -c /tmp/dnsmasq.d/.blocklist.gz\nexit 0\n" > /tmp/dnsmasq.d/.extract_blocklist
-		gzip -f /tmp/blocklist
-		mv /tmp/blocklist.gz /tmp/dnsmasq.d/.blocklist.gz
-		imported_blocklist_file_size_KB=$(get_file_size_KB /tmp/dnsmasq.d/.blocklist.gz)
+		printf "conf-script=\"busybox sh /var/run/adblock-lean/dnsmasq.d/.extract_blocklist\"\n" > /var/run/adblock-lean/dnsmasq.d/conf-script
+		printf "busybox gunzip -c /var/run/adblock-lean/dnsmasq.d/.blocklist.gz\nexit 0\n" > /var/run/adblock-lean/dnsmasq.d/.extract_blocklist
```

Comment:
I think dnsmasq looks for conf-script in /tmp/dnsmasq.d/ ? I'm still not really sure how this works but that's the impression I got.

Reply:
I need to fix this too!